### PR TITLE
Create hi_IN

### PR DESCRIPTION
**Pull request by Probattler**
Hindi translation of the description.

### CODE REVIEW

**Review by osfanbuff63 — COMMENTED** [review]
Add a `.md` to the file name, but looks good

**Comment by Probattler** [comment]
Oh forgot about it.

File: Description/hi_IN.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Hindi (India)
```
Comment (Madis0) [inline Description/hi_IN.md]: You can write that in Hindi as well.

File: Description/hi_IN.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Hindi (India) 
+  
+ एक Fabric-based modpack Minecraft के लिए जो परफॉर्मांस और ग्राफिक्स को बेहतर करने पर अपना ध्यान केंद्रित करता है।
```
Comment (Madis0) [inline Description/hi_IN.md]: None of those 4 words have local equivalents?

File: Description/hi_IN.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Hindi (India) 
+  
+ एक Fabric-based modpack Minecraft के लिए जो परफॉर्मांस और ग्राफिक्स को बेहतर करने पर अपना ध्यान केंद्रित करता है। 
+  
+ [![अभी की समीक्षा](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I) 
+  
+ ### परफॉर्मांस
+  
+ Fabulously Optimized ज्यादतर Optifine से बेहतर साबित होता है क्योंकि उसमे [आधुनिक पेरफॉर्मांस मॉड्स का इस्तेमाल किया गया है][1] जैसे Sodium और Lithium.
```
Comment (Madis0) [inline Description/hi_IN.md]:
```suggestion
 Fabulously Optimized ज्यादतर OptiFine से बेहतर साबित होता है क्योंकि उसमे [आधुनिक पेरफॉर्मांस मॉड्स का इस्तेमाल किया गया है][1] जैसे Sodium और Lithium. 
```

File: Description/hi_IN.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  
+ [![अभी की समीक्षा](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I) 
+  
+ ### परफॉर्मांस
+  
+ Fabulously Optimized ज्यादतर Optifine से बेहतर साबित होता है क्योंकि उसमे [आधुनिक पेरफॉर्मांस मॉड्स का इस्तेमाल किया गया है][1] जैसे Sodium और Lithium. 
+  
+ ### OptiFine के साथ समानता 
+  
+ Fabulously Optimized [Optifine के लगभग सभी विभिन्न विशेषताओं को स्पोर्ट करता है][2], जिसमे shaders, connected textures और zoom आते हैं. [एक मुफ्त cape पाएं!][3]
```
Comment (Madis0) [inline Description/hi_IN.md]:
```suggestion
 Fabulously Optimized [OptiFine के लगभग सभी विभिन्न विशेषताओं को स्पोर्ट करता है][2], जिसमे shaders, connected textures और zoom आते हैं. [एक मुफ्त cape पाएं!][3] 
```

File: Description/hi_IN.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  
+ Fabulously Optimized ज्यादतर Optifine से बेहतर साबित होता है क्योंकि उसमे [आधुनिक पेरफॉर्मांस मॉड्स का इस्तेमाल किया गया है][1] जैसे Sodium और Lithium. 
+  
+ ### OptiFine के साथ समानता 
+  
+ Fabulously Optimized [Optifine के लगभग सभी विभिन्न विशेषताओं को स्पोर्ट करता है][2], जिसमे shaders, connected textures और zoom आते हैं. [एक मुफ्त cape पाएं!][3] 
+  
+ ### कम बदलाव 
+  
+ Fabulously Optimized [मे कुछ बदलाव हैं][4] जो vanilla भावों को बिल्कुल नहीं बदलता, जैसे ज्यादा chat history, windowed full screen mode और bugfixes सबसे आम मुद्दों के लिए.
```
Comment (Madis0) [inline Description/hi_IN.md]: Really? Come on, these words can't be untranslatable...

File: Description/hi_IN.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+ > **A**: नहीं, ये किसी भी सर्वर पर काम करता है जो vannila clients को support करते हैं. [ज्यादा विस्तृत मे उत्तर और सेर्वर से जुड़े टिप्स यहाँ मौजूद हैं.][13]  
+ >  
+ > [![](https://i.ibb.co/gr9mSxW/image.png) अगर आपको एक सेर्वेर होस्ट चाहिए, यहाँ दबाएं- 25 प्रतियों डिस्कॉउट पहले महीने के लिए!][14] 
+  
+  
+ > **Q**: मेरे पास आर भी प्रशन हैं! 
+ >  
+ > **A**: [आयें और हमारे discord पर पूछे!][10] 
+  
+ ### इस modpack को उस करके, [आप हमारे following disclaimers से सहमत होते हैं][15].
```
Comment (Madis0) [inline Description/hi_IN.md]: come on

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: Description/hi-IN.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
+# Hindi (India)
+
+एक Fabric-based modpack Minecraft के लिए जो परफॉर्मांस और ग्राफिक्स को बेहतर करने पर अपना ध्यान केंद्रित करता है।
+
+[![अभी की समीक्षा](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```
Comment (Madis0) [inline Description/hi-IN.md:5]:
```suggestion
[![अभी की समीक्षा](https://img.youtube.com/vi/K90gsbmhf3w/maxresdefault.jpg)](https://www.youtube.com/watch?v=K90gsbmhf3w)
```
A review in Hindi

File: Description/hi-IN.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+> **Q**: कुछ टूट गया! तुम्हे कुछ जोड़ना है!
+> 
+> **A**: [हमारे GitHub पेज पर डालें.][8]
+
+
+> **Q**: कौनसे mods उपलब्ध हैं? 
+> 
+> **A**: किसी भी संस्करण को दबाकर नीचे जाएं [या नीचे दी गयी टेबल देखें][12].
+
```
Comment (Madis0) [inline Description/hi-IN.md:54]: There is a new entry here in upstream btw

File: Description/hi-IN.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+> **प्रशन**: मुझे ये अच्छा लगा. में इसे बनाने मे कैसे मदत कर सकता हूँ?
+> 
+> **उत्तर**: [मदत करने के तरीके यहाँ पर दिये हुए हैं.][16] शुक्रिया!
+
+
+> **प्रशन**: क्या मुझे ये मोडपैक अपने सर्वर पे भी चाहिए होगा?
+> 
+> **उत्तर**: नहीं, ये किसी भी सर्वर पर काम करता है जो vannila clients को support करते हैं. [ज्यादा विस्तृत मे उत्तर और सेर्वर से जुड़े टिप्स यहाँ मौजूद हैं.][13]
```
Comment (Madis0) [inline Description/hi-IN.md:63]: > vannila clients को support 

fix vanilla* typo at least, but I believe at least "support" can be translated too

**Comment by Madis0** [comment]
Thanks!

**Pull request merged**